albatross-daemon fails to restore unikernels because fifo directory does not exist

```
albatrossd[499]: albatrossd: [ERROR] failed to create [vm: resolver]: file /run/albatross/fifo/resolver error in mkfifo: No such file or directory
```
The fifo directory is created (only) by the albatross_console.service systemd script. I am not sure why it is there and not in albatross_daemon.service.
https://github.com/roburio/albatross/blob/340f0e4771b57e50f4d7ff63d3115f353db50527/packaging/Linux/albatross_console.service#L15-L17

--- a/packaging/Linux/albatross_console.service
+++ b/packaging/Linux/albatross_console.service
@@ -10,11 +10,6 @@ After=syslog.target
 Type=simple
 User=albatross
 ExecStart=/usr/libexec/albatross/albatross-console --systemd-socket-activation --tmpdir="%t/albatross/" -vv
-RuntimeDirectoryPreserve=yes
-RuntimeDirectory=albatross
-ExecStartPre=/bin/mkdir -p %t/albatross/fifo
-ExecStartPre=/bin/chmod 2770 %t/albatross/fifo
-ExecStartPre=/bin/mkdir -p %t/albatross/util
 PIDFile=%t/albatross/console.pid
 RestrictAddressFamilies=AF_UNIX
 

--- a/packaging/Linux/albatross_daemon.service
+++ b/packaging/Linux/albatross_daemon.service
@@ -12,6 +12,11 @@ ExecStart=/usr/libexec/albatross/albatrossd --systemd-socket-activation --tmpdir
 #RuntimeDirectoryPreserve=yes
 #RuntimeDirectory=albatross
 PIDFile=%t/albatross/daemon.pid
+RuntimeDirectoryPreserve=yes
+RuntimeDirectory=albatross
+ExecStartPre=/bin/mkdir -p %t/albatross/fifo
+ExecStartPre=/bin/chmod 2770 %t/albatross/fifo
+ExecStartPre=/bin/mkdir -p %t/albatross/util
 
 ## hardening stuff
 ProtectSystem=full

